walk-in-interview/src/main/java/com/google/job/servlets/JobServlet.java

```diff
@@ -44,7 +46,17 @@ public void service(HttpServletRequest request, HttpServletResponse response) th
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
just make sure: here need to throw IOException? or simply add that in the catch block?

Reply:
yep, added it to the catch block